lxd/apparmor/instance_qemu.go

```diff
@@ -75,7 +75,7 @@ profile "{{ .name }}" flags=(attach_disconnected,mediate_deleted) {
 
   # Snap-specific paths
   /var/snap/lxd/common/ceph/**                         r,
-  /var/snap/microceph/current/conf/**                  r,
+  /var/snap/microceph/*/conf/**                        r,
```

Comment:
`current` feels weird as it's AFAIK, always a symlink pointing to the current revision. However, I would expect the conf to live under `common` in which case we could have a more specific rule. @stgraber any reason not to have that conf file in `common`?

Reply:
For microceph, we're storing the configuration in the versioned path so that it's possible to compare it across refreshes.

The commit looks correct to me as `current` being a symlink won't work as apparmor always derefs the symlinks prior to applying policy.